framework/logresource/resource.go

```diff
@@ -61,66 +61,66 @@ func New(config Config) (*Resource, error) {
 }
 
 func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
-	r.logger.Log("debug", PreOperationMessage, "operation", "GetCurrentState")
+	r.logger.Log("action", "start", "component", "operatorkit", "function", "GetCurrentState")
```

Comment:
I'd call `function` a `func` to express being a Go funboy :D

Reply:
I don't. We also went with `function` in the framework logs.